src/main/java/org/folio/search/service/setter/bibframe/authority/BibframeAuthorityLccnProcessor.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.search.service.setter.bibframe.authority;
+
+import static java.util.stream.Collectors.toCollection;
+import static org.folio.search.domain.dto.BibframeAuthorityIdentifiersInner.TypeEnum.LCCN;
+
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.folio.search.domain.dto.BibframeAuthority;
+import org.folio.search.domain.dto.BibframeAuthorityIdentifiersInner;
+import org.folio.search.service.lccn.LccnNormalizer;
+import org.folio.search.service.setter.FieldProcessor;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class BibframeAuthorityLccnProcessor implements FieldProcessor<BibframeAuthority, Set<String>> {
+
+  @Qualifier("lccnNormalizerStructureB")
```

Comment:
Hi @aleksei-pronichev, We have to use the DefaultLccnNormalizer here. 
lccnNormalizerStructureB is only applicable for Bib records acquired after 2001